dialer.go

```diff
@@ -36,7 +36,24 @@ const (
 	serverProxyPort = "3307"
 )
 
+var (
+	// defaultKeys is the default pub/priv encryption keypair for the client, and is
```

Comment:
Might as well be consistent and make this "is the default rsa.PrivateKey representing the client..."